pandas/io/formats/style.py

```diff
@@ -879,7 +891,7 @@ def set_caption(self, caption):
         self.caption = caption
         return self
 
-    def set_table_styles(self, table_styles):
+    def set_table_styles(self, table_styles: List) -> "Styler":
```

Comment:
can we say anything about what goes in the List?

Reply:
(same question for other occurrences of List above)

Reply:
The docs below it says it should be a Dict.

So I am even more confused.

Reply:
When trying to annotate it as ```List[str]```

mypy gives this error:

```
pandas/io/formats/style.py:861: error: Argument 1 to "extend" of "list" has incompatible type "List[str]"; expected "Iterable[Tuple[Callable[..., Any], Tuple[Any, ...], Dict[Any, Any]]]"
Found 1 error in 1 file (checked 794 source files)
````

> expected "Iterable[Tuple[Callable[..., Any], Tuple[Any, ...], Dict[Any, Any]]]"

Maybe I should leave it unannotated, or leave it as ```List```, or put ```Any``` just in case?